test/container_common_test.cpp

```diff
@@ -987,6 +987,22 @@ TEST_CASE("container_common_test, set")
     REQUIRE_EQ(set_symmetric_difference(intSet1, intSet2), intSet7);
     REQUIRE_EQ(set_intersection(intSet1, intSet2), intSet4);
     REQUIRE_EQ(sets_intersection(IntSets({intSet1, intSet2, intSet3})), intSet8);
+
+    typedef std::unordered_set<int> IntUnordSet;
+
+    IntUnordSet intUnordSet1 = {0,1,2,3};
+    IntUnordSet intUnordSet2 = {2,3,4,5};
+    IntUnordSet intUnordSet3 = {0,2};
+    IntUnordSet intUnordSet4 = {2,3};
+    IntUnordSet intUnordSet5 = {0,1};
+    IntUnordSet intUnordSet6 = {0,1,2,3,4,5};
+    IntUnordSet intUnordSet7 = {0,1,4,5};
+
+    REQUIRE_EQ(unordered_set_includes(intUnordSet1, intUnordSet3), true);
```

Comment:
If you would use a literal construction here, the code reader doesn't have to skip his focus up and down to verify the test requirement.  On top of that, you don't have to think deeply 'is `intUnordSet7` even used?'.

Also, is it possible to test for trueness and falseness instead of equality?

    REQUIRE_FALSE( unordered_set_includes(Unordered{0, 2}, {0, 1, 2, 3}) );